rasa_sdk/interfaces.py

```diff
@@ -271,6 +272,29 @@ def add_slots(self, slots: List[EventType]) -> None:
             self.slots[event["name"]] = event["value"]
             self.events.append(event)
 
+    def get_intent_if_not_fallback(self) -> Optional[Text]:
+        """Retrieves the intent the last user message would have been classified as
+        if it was not nlu_fallback
+        """
+        last_user_action = self.get_last_event_for("user")
+        if not last_user_action:
+            warnings.warn("No last user action.")
+            return None
+
+        parse_data = last_user_action["parse_data"]
```

Comment:
There should always be a `parse_data`, but I think this would be more robust (who knows what the API gets fed)
```suggestion
        parse_data = last_user_action.get("parse_data", {})
```